hazelcast/src/main/java/com/hazelcast/map/impl/DataLimitExceededException.java

```diff
@@ -0,0 +1,7 @@
+package com.hazelcast.map.impl;
+
+import com.hazelcast.core.HazelcastException;
+
+public class DataLimitExceededException
```

Comment:
Isn't it query result size exceeded? Data limit exceeded makes my think of my mobile internet provider.